src/MarkovModel.py

```diff
@@ -160,7 +160,7 @@ def detNewState(self,detTrans):
       elif len(detTrans[key]) == 2:
         lowVal  = min(detTrans[key])
         highVal = max(detTrans[key])
-        time = self.randomEngine.uniform(low=lowVal, high=highVal)
+        time = self.randomEngine._engine.uniform(low=lowVal, high=highVal)
```

Comment:
It might possibly be better to use the scipy uniform (and exponential and choice) random number generators directly, instead of using the _engine in randomEngine. That said, this way works.